document-parsers/langchain4j-document-parser-llamaparse/src/main/java/dev/langchain4j/data/document/parser/llamaparse/LlmaParseApi.java

```diff
@@ -0,0 +1,57 @@
+package dev.langchain4j.data.document.parser.llamaparse;
+
+import okhttp3.MultipartBody;
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Headers;
+import retrofit2.http.Multipart;
+import retrofit2.http.POST;
+import retrofit2.http.Part;
+import retrofit2.http.Path;
+
+interface LlmaParseApi {
+
+    enum JOB_STATUS {
+        PENDING("PENDING"),
+        SUCCESS("SUCCESS"),
+        ERROR("ERROR"),
+        CANCELED("CANCELED");
+
+        private String value;
+        JOB_STATUS(String value) {
+            this.value = value;
+        }
+
+        String value() {
+            return this.value;
+        }
+    }
+
+    final String baseUrl = "https://api.cloud.llamaindex.ai/api/parsing/";
```

Comment:
once `baseUrl` in `LlamaParseClient` has a default value, you can remove it from here